### PR TITLE
add more tips for PR reviews

### PR DESCRIPTION
**Pull request by hayleypixie**
Added some things I've learned about PR reviews.

Let me know your thoughts. Happy to discuss/remove/change things.

### CODE REVIEW

File: 03-project-delivery/06-reviews.md [hunk 1]
```diff
@@ -3,18 +3,48 @@
...
+  - Written feedback is often read in an overly critical tone in our heads.
+  - We’re all friends here. How would the reviewer be saying this to you in person?
+- Try to respond to every comment and resolve them as you go.
+  - Link to specific commits - "Well spotted! Done in 123987"
+- Provide links and learning resources that you found useful so the reviewer
+can also benefit from your research.
+  - If you have needed to look something up, the reviewer might too.
+  - Put links in the description to tutorials & articles that helped you.
+  - Leave links to useful documentation as comments in the code and future devs
+  will also benefit.
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:47]:
```suggestion
  can also benefit.
```

File: 03-project-delivery/06-reviews.md [hunk 2]
```diff
@@ -26,7 +56,39 @@
...
+- Sharing knowledge avoids silos and any comment is a learning opportunity
+  for both so ask questions.
+- Be humble.
+  - Assume the reviewee has the best understanding of the problems being
+  solved, they’ve put a lot of thought and work into it.
+  - Trust in their decisions but ask for clarification.
+- Be aware that written feedback is often read in an overly critical tone.
+  - Prefer open-ended questions over making strong or opinionated statements.
+  - Prefer we instead of you.
+    - "-another strategy- would be better here."
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:70]:
```suggestion
    - bad: "-another strategy- would be better here."
```

File: 03-project-delivery/06-reviews.md [hunk 3]
```diff
@@ -26,7 +56,39 @@
...
+    - better: "We should use -another strategy- because -a reason-."
+    - great: "Could we use -another strategy- for -a reason-? Why did you chose
+    this way?"
+- If there’s a lot of feedback:
+  - Consider what are the must dos and what, if anything, can be left out to
+  avoid over-facing someone & keep us progessing.
+    - Must dos - the important stuff. Highlight them in a change request.
+    - Should dos - make a task/ticket if anything can be done as later.
+    - Could dos - "We could do -suggestion-. Not important, you decide."
+- Check the [handbook](./03-style-guidelines.md) for things that are to do with
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:80]:
```suggestion
- Use the [handbook](./03-style-guidelines.md) as a code style reference.
```

(and delete next line)

File: 03-project-delivery/06-reviews.md [hunk 4]
```diff
@@ -26,7 +56,39 @@
...
+code style.
+  - We should be on the same page with our code style, if not, prefer to discuss as
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:82]:
```suggestion
  - We should be on the same page with our code style, if not, we should discuss as
```

File: 03-project-delivery/06-reviews.md [hunk 5]
```diff
@@ -26,7 +56,39 @@
...
+  a team rather than between individuals on PR’s.
+  - If it’s not in the handbook or you disagree with something, bring it up in
+  retro & then update the handbook once we have agreement.
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:85]:
```suggestion
  retro and then update the handbook once we have agreement.
```

File: 03-project-delivery/06-reviews.md [hunk 6]
```diff
@@ -26,7 +56,39 @@
...
+  - As time goes on we'll all know what's expected and these comments will
+  disappear from PRs - speed bonus :D
+- Do give praise for good work :)
+  - Highlight things you liked
+  - Highlight things you’ve learned
+  - ::taco::
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:91]:
```suggestion
  - Give an appreciation ::taco:: in #appreciations on Slack!
```

File: 03-project-delivery/06-reviews.md [hunk 7]
```diff
@@ -51,6 +113,7 @@
 - Rails: Does brakeman report any new security issues?
 - Asset pipeline JavaScript: Have any ES6-isms snuck in e.g. const/let, arrow
   functions? This is OK if the JS is being compiled by Webpacker or similar.
+- React: Are there any missing PropTypes?
```
Comment (jalada) [inline 03-project-delivery/06-reviews.md:116]: Maybe add a link to the proptypes library?

**Review by jalada — CHANGES_REQUESTED** [review]
This is some great added resolution @hayleypixie, nice one! I may spot some other things once its live on the site but this is a great start.

**Comment by hayleypixie** [comment]
Maybe we can add something about target pull request sizes?

**Comment by jalada** [comment]
> Maybe we can add something about target pull request sizes?

We actually already have something for that under [Project Delivery Metrics](https://handbook.pixielabs.io/#/02-project-management/05-project-delivery-metrics) but perhaps a link to that somewhere pertinent?

**Comment by hayleypixie** [comment]
@jalada is this good to merge now?

**Comment by jalada** [comment]
Do it!

**Pull request merged**